homeworks/01-bubble-sort/src/components/button/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from "react";
+
+type ButtonProps = {
+	name: string,
+	isDisabled: boolean,
+	action: () => void,
+}
+
+class Button extends Component<ButtonProps, {}> {
```

Comment:
The second type is optional: `Component<ButtonProps>`

Reply:
fixed